src/main/scala/org/broadinstitute/hail/check/Gen.scala

```diff
@@ -208,27 +208,27 @@ object Gen {
     * elements in finite time.
     */
   def distinctBuildableOfAtLeast[C[_], T](min: Int, g: Gen[T])(implicit cbf: CanBuildFrom[Nothing, T, C[T]]): Gen[C[T]] =
-    Gen { (p: Parameters) =>
-      val b = cbf()
-      if (p.size < min) {
-        throw new RuntimeException(s"Size (${ p.size }) is too small for buildable of size at least $min")
-      } else if (p.size == 0)
-        b.result()
-      else {
-        val s = p.rng.nextInt(min, p.size)
-        val part = partition(p.rng, p.size, s)
-        val t = mutable.Set.empty[T]
-        for (i <- 0 until s) {
-          var element = g.resize(part(i))(p)
-          while (t.contains(element)) {
-            element = g.resize(part(i))(p)
-          }
-          t += element
+  Gen { (p: Parameters) =>
```

Comment:
I'm happy to just merge (this is only formatting) but it looks like the `Gen {` has the same indentation as the function declaration above it, which seems wrong.  Maybe try rebooting intellij and see if that fixes it?  Otherwise let's put this in and rebuild